src/main/java/com/google/adwords/KenshooSelectorBuilder.java

```diff
@@ -0,0 +1,31 @@
+package com.google.adwords;
+
+import sync.google.KenshooSelector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by user on 12/28/2016.
+ */
+public class KenshooSelectorBuilder {
+
+        private List<Integer> campaignIdInTargets;
+
+        public KenshooSelectorBuilder() {
+        }
+
+        public KenshooSelectorBuilder campaignIdInTargets(List<Integer> val) {
+            campaignIdInTargets = val;
+            return this;
+        }
+
+        public KenshooSelectorBuilder  withCampaignIdInTarget(Integer val) {
+            campaignIdInTargets.add(val);
```

Comment:
better to reset the campaignIdInTarget field rather than adding an element.
for instance, if your builder also had a asCloneOf() method:
```
    Selector newSelector = Builder.asCloneOf(oldSelector).withCampaignIds(2, 4, 6).build();
```
When I read it, I expect newSelector to be a clone of oldSelector but with campaign ID's 2, 4, 6.
Instead, it will have the old campaign ID's in addition.